node/buffer.ts

```diff
@@ -245,6 +246,30 @@ export class Buffer extends Uint8Array {
     );
   }
 
+  boundsError(value: number, length: number, type: string): never {
```

Comment:
This exposes `boundsError` in Buffer class, which doesn't exist in Node.js version. So I'd suggest we should inline these in `readUIntBE`

Reply:
I will make it private and not inline it on `readUIntBE`, because we will use it for `readUIntLE`.
I will Open PR for `readUIntLE` later.